common/elasticsearch/src/test/scala/uk/ac/wellcome/elasticsearch/test/fixtures/ElasticsearchFixtures.scala

```diff
@@ -241,7 +251,22 @@ trait ElasticsearchFixtures
         }
         r
       }
+  }
 
+  def indexObjectCompressed[T](index: Index, t: T)(
+    implicit encoder: Encoder[T]): Future[Response[IndexResponse]] = {
+    val doc = toJson(t).get
+    debug(s"ingesting: $doc")
+    elasticClientWithCompression
+      .execute {
+        indexInto(index.name).doc(doc)
+      }
+      .map { r =>
+        if (r.isError) {
+          error(s"Error from Elasticsearch: $r")
+        }
+        r
+      }
```

Comment:
I would inline this method and the definition of elasticClientWithCompression in the test case, so it's easier to see what that test doing. It seems unlikely we'd reuse this client. Otherwise, looks good.

Reply:
I think fine as the `elasticClientWithCompression` is lazy, and we don't want to recreate the client lots of times if for example the call to this method was wrapped in an `eventually`